src/views/AddPriceSingle.vue

```diff
@@ -412,18 +412,28 @@ export default {
     showUserRecentProofs() {
       this.userRecentProofsDialog = true
     },
-    handleProofSelected(proofId) {
-      this.addPriceSingleForm.proof_id = proofId
+    handleProofSelected(proof) {
```

Comment:
the problem here is that above in `mounted` we only have the proof_id, so this will not work...

Reply:
can be fixed by keeping `proofId`, and removing the date init.

Reply:
and add an extra param to fetch or not the proof image ? because if coming from the `UserRecentProofsDialog` then it we be called even though we already have the image (line 424)

Reply:
Oh, I forgot to change the function used in `mounted` to `getProofById` when copying the code over to GitHub. Did that now.

Reply:
Not sure what you mean by it being called twice when using the `UserRecentProofsDialog` though. It's only calling it once for me. Maybe I'm misunderstanding you or something is wrong with my testing environment. I couldn't get image uploading to work, uploaded pics just give me 404 error when I try to access the upload path, so I had to do a few tweaks to work around that by using remote URLs instead.

Reply:
my bad there doesn't seem to be duplicate loading :ok_hand: